public/src/components/channelManagement/MutliSelectTagEditor.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import { makeStyles } from '@mui/styles';
+import { grey } from '@mui/material/colors';
+import { Theme } from '@mui/material/styles';
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  container: {
+    width: '100%',
+    '& > * + *': {
+      marginTop: spacing(3),
+    },
+    borderColor: `2px solid ${{ color: grey[700] }}`,
+    borderRadius: '2px',
+    padding: spacing(2),
+  },
+}));
+
+interface Option {
+  label: string;
+  value: string;
+}
+
+const options: Option[] = [
+  { label: 'environment/climate-change', value: 'environment/climate-change' },
```

Comment:
where did the list of different tag options come from? is it in a Trello ticket?

Reply:
The tags we track are mentioned [here](https://github.com/guardian/support-dotcom-components/blob/main/packages/shared/src/lib/history.ts#L64) in SDC

Reply:
thanks!